tests/util/caches/test_deferred_cache.py

```diff
@@ -26,7 +26,7 @@ class DeferredCacheTestCase(TestCase):
     def test_empty(self) -> None:
         cache: DeferredCache[str, int] = DeferredCache("test")
         with self.assertRaises(KeyError):
-            cache.get("foo")
+            cache.get("foo")  # type: ignore[unused-awaitable]
```

Comment:
AFAICS  `cache` is roughly a clever `Mapping[str, Deferred]` throughout this file, and we're checking that we can get and set its contents. It's okay that we're not awaiting on them. But please double-check this.